### PR TITLE
Prepare templates and js files for localization

### PR DESCRIPTION
**Pull request by brew**
This PR prepares the codebase for localization by:

- fixing POT generation within the gulp pipeline
- marking strings with `gettext` in templates
- marking strings with `gettext` in js files
- generating a new POT file

This partly addresses #844.

### CODE REVIEW

File: census/middlewares/index.js [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
     });
 };
 
 var internalServerError = function(err, req, res, next) {
   console.error(err.stack);
   res.status(500)
     .render('500.html', {
-      title: '500: Internal Server Error',
-      message: 'Something is up with the server.<br />' + err
+      title: req.gettext('500: Internal Server Error'),
```
Comment (akariv) [inline census/middlewares/index.js]: Are you sure you want to complicate a 500 error page?

Comment (brew) [inline census/middlewares/index.js]: In case there's a problem with `req.gettext` which results in an endless 500?

Comment (akariv) [inline census/middlewares/index.js]: For example.
Also, where does the language information come from?

**Comment by akariv** [comment]
Apart from one comment for consideration, LGTM.
I didn't check that _all_ strings were converted, though.

File: census/middlewares/index.js (hunk 1 above)
Comment (brew) [inline census/middlewares/index.js]: Request middleware, provided by i18n-abide.

**Comment by brew** [comment]
Thanks!

**Pull request merged**